source/postprocess/gravity_point_values.cc

```diff
@@ -414,6 +432,47 @@ namespace aspect
             }
         }
 
+      // write quantities in the statistic file
+      const std::string name0("Stokes velocity polynomial degree");
+      statistics.add_value (name0, degree-quadrature_degree_increase);
+      statistics.set_precision (name0, 0);
+      statistics.set_scientific (name0, false);
+
+      const std::string name1("Quadrature degree increase for gravity calculation");
+      statistics.add_value (name1, quadrature_degree_increase);
+      statistics.set_precision (name1, 0);
+      statistics.set_scientific (name1, false);
+
+      const std::string name2("Average gravity (m/s^2)");
+      statistics.add_value (name2, sum_g/n_satellites);
+      statistics.set_precision (name2, 18);
```

Comment:
Do you really need 18 digits of precision? First of all `double` will only accurately represent 15 digits anyway, and that is only in case there is no loss of precision due to calculations and no model (iterative solvers, limited resolution) error at all. Our usual accuracy is 8 digits. Some quantities require higher precision and they usually use 12 digits. I realize gravity variations can be very small compared to the full gravity, but have you tested that you need more than 12 digits?

Reply:
I chose 18 digits based on early tests I made for the graph below. I remember I picked 8, 12, 14 then jumped to 18 digits, unaware that the double was limited to 15. It was useful for a benchmark, but I guess for users it is pointless (even perhaps for benchmarks) to have that high accuracy/precision since there is a clear convergence of the solution way below the milligal threshold. 
![image](https://user-images.githubusercontent.com/32137895/77296541-24924680-6ce8-11ea-90f6-243715b6ebab.png)

Reply:
In the output of the gravity postprocessor, should I also change the precision to 12 digits?